cms/templates/maintenance/_force_publish_course.html

```diff
@@ -0,0 +1,80 @@
+<%page expression_filter="h"/>
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML, Text
+%>
+<div class="wrap-instructor-info studio-view maintenance-form">
+    <form class="form-create" method="post">
+        <input type="hidden" name="csrfmiddlewaretoken" value="${csrf_token}"/>
+        <div class="wrapper-form">
+            <fieldset>
+                <ol class="list-input">
```

Comment:
Is there a particular reason we are using the ordered list element here?  This doesn't seem to be a numbered list of items, especially since there are no visual numbers showing.  We should only be using lists to markup actual lists.